src/OrchardCore/OrchardCore.Mvc.Core/SharedViewCompilerProvider.cs

```diff
@@ -28,6 +33,7 @@ public class SharedViewCompilerProvider : IViewCompilerProvider
         private readonly ApplicationPartManager _applicationPartManager;
         private readonly IRazorViewEngineFileProviderAccessor _fileProviderAccessor;
         private readonly CSharpCompiler _csharpCompiler;
+        //private readonly IViewCompilationMemoryCacheProvider _compilationMemoryCacheProvider;
```

Comment:
Remove all the comments

Reply:
Okay, not on this branch, i will do very soon.